.github/workflows/tests.yml

```diff
@@ -7,7 +7,27 @@ on:
     branches: [ master ]
 
 jobs:
+
+  check_code_quality:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.7"
```

Comment:
see my comment below about considering v3.8 as the minimum version since this one will be eol in 6 months

Reply:
sounds good